lml/src/main/java/com/github/czyzby/lml/parser/impl/attribute/group/DebugRecursivelyLmlAttribute.java

```diff
@@ -2,21 +2,37 @@
 
 import com.badlogic.gdx.scenes.scene2d.Group;
 import com.github.czyzby.lml.parser.LmlParser;
+import com.github.czyzby.lml.parser.action.ActorConsumer;
 import com.github.czyzby.lml.parser.tag.LmlAttribute;
 import com.github.czyzby.lml.parser.tag.LmlTag;
+import com.github.czyzby.lml.util.LmlUtilities;
 
 /** See {@link Group#setDebug(boolean, boolean)}. Second method argument is always true, so this method sets debug for
  * all children; for non-recursive debug, use default debug attribute. Mapped to "debugRecursively".
  *
  * @author MJ */
 public class DebugRecursivelyLmlAttribute implements LmlAttribute<Group> {
+    private static class DebugAction implements ActorConsumer<Object, Object> {
```

Comment:
Yeah, I might have mixed it up with Java 8 `Function`. Now that I think about it, the generics order could have been better.

`LmlUserObject` is pretty much internal API, and I may have chosen some questionable designs to escape Java's generics and reflection hell. I'll accept this PR as is (once I have some time to test it - I'm currently at work), and I'll see if this can be achieved without `ActorConsumer` altogether.